incubator/nodejs/README.md

```diff
@@ -1,56 +1,36 @@
 # Node.js Stack
 
-A stack is the foundation of your application. This stack provides a prototype platform definition for Node.js v10 based projects.
+The Node.js stack provides a consistent way of developing [Node.js](https://nodejs.org/) applications. As an asynchronous event-driven JavaScript runtime, Node is designed to build scalable network applications.
 
-## Templates
+This stack is based on `Node.js v10` runtime and allows you to bring any Node.js application that can work with this version of the runtime.
```

Comment:
Can you add "the" in front of `Node.js v10`?
What do we mean by the word 'bring'?

Reply:
> addressed